### PR TITLE
Add alertmanager, Telegram alerting

### PR DESCRIPTION
**Pull request by hhio618**
### Description
This will add:
- Prometheus Alertmanager
- A Telegram Alertmanager receiver using [this repo](https://github.com/metalmatze/alertmanager-bot)

### CODE REVIEW

File: configs/manifests/alerting-persist.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
```
Comment (krasi-georgiev) [inline configs/manifests/alerting-persist.yml:2]: What would we loose if we don't use persistent volume?

File: docs/setup-and-usage.md [hunk 2]
```diff
@@ -105,6 +105,17 @@ kubectl apply -f configs/manifests/monitoring-persist.yml
 kubectl apply -f configs/manifests/monitoring.yml
 ```
 
+* Optionally deploy the alerting manager and get alerts on Telegram.
```
Comment (krasi-georgiev) [inline docs/setup-and-usage.md:108]: Maybe also add a link to the bot readme for how to setup. IIRC there was some additional setup right?

File: configs/manifests/alerting-persist.yml (hunk 1 above)
Comment (hhio618) [reply configs/manifests/alerting-persist.yml:2]: It is needed for saving list of chat that the bot added to them. see [here.](https://github.com/metalmatze/alertmanager-bot/blob/11438e166eac991c6540e10dcfdcabcb2006e349/pkg/telegram/chats.go)

File: docs/setup-and-usage.md (hunk 2 above)
Comment (hhio618) [reply docs/setup-and-usage.md]: I added their repo url. hopefully no additional setup required for bot to be working.

File: configs/manifests/alerting-persist.yml (hunk 1 above)
Comment (krasi-georgiev) [reply configs/manifests/alerting-persist.yml:2]: ok lets make it really small as I don't think it will use much space. something like 100MB

**Review by themandalore — APPROVED** [review]

**Pull request merged**